src/backends/smtlib2.rs

```diff
@@ -14,6 +14,7 @@ use petgraph::EdgeDirection;
 
 use backends::backend::{Logic, SMTBackend, SMTError, SMTNode, SMTResult};
 use theories::{bitvec, core, integer};
+use super::backend::SMTRes;
```

Comment:
Yes, this may be a relic of how I developed it. This probably will have to change.